### PR TITLE
Added FAQs

### PR DESCRIPTION
**Pull request by NikhilR129**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -193,6 +194,49 @@
...
+                </div>
+            </div>
+              <div class="faq-content">
+                <div class="question">
+                    <p>
+                        <strong>Do you provide any travel reimbursements for the event?</strong>
+                    </p>
+                </div>
+                <div class="answer">
+                    <p>Not quite. But in certain cases where we find the profile of a developer/team to be extra-ordinary,we may consider it again.In any case,we will need valid proof that you cannot afford travelling to Allahabad.Contact us in such a case.</p>
```
Comment (gauthamzz) [inline index.html]: @hackerkid you said no

Comment (NikhilR129) [inline index.html]: Should I change it?

Comment (gauthamzz) [inline index.html]: idk let him reply over here

Comment (NikhilR129) [inline index.html]: Merge it.I will send a PR again,if he says no.

Comment (hackerkid) [inline index.html]: @gauthamzz Looks good to me. :+1:

**Pull request merged**